docs/billing-and-plans/cancel-active-paid-plan.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Cancel Active Paid Subcription Plan
+intro: Sometimes the project needs may change and the current active plan does not fit your project needs. Cancel your currentactive plan to ensure you will not be charged further.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, in the upper-right corner of any page, click your profile icon then click `Billing & plans`
+    > This page shows the current subscription plan information including: plan name, credit limit, remaining credits and expiration date.
+1. Click `MANAGE YOUR PLAN`
+    > This will open your billing page
+1. In `Current subscription` section, click `Cancel my subscription` and follow the prompts
```

Comment:
Our docs used to mention that in a separate step, to make it easier for the reader to digest the info: https://github.com/search?q=repo%3Adevopness%2Fdevopness%20prompts&type=code

Are you here suggesting a new approach for instructing users on how to follow the prompts and ditch the current pattern? 🤔 

Benefits?

Reply:
```suggestion
1. In `Current subscription` section, click `Cancel my subscription`
1. Follow the prompts to complete the subscription cancellation
```

@jfoliveira agreed, would this be better aligned with the Devopness tone?

Reply:
Better!